README.md

```diff
@@ -14,23 +14,32 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+### Setting Local Environment Variables
+
+When running locally on a development machine you can manually set the application's configuration through a `.env` file at the root of the source tree:
+
+```env
+NEXT_PUBLIC_BACKEND_SERVICE_PROTOCOL="http"
+NEXT_PUBLIC_BACKEND_SERVICE_PORT=4000
+NEXT_PUBLIC_BACKEND_SERVICE_HOST="localhost"
+NEXT_PUBLIC_BACKEND_API_VERSION="0.0.1"
+
+# TIPTAP_APP_ID originates from your TipTap Cloud Dashboard
+NEXT_PUBLIC_TIPTAP_APP_ID="<TIPTAP_APP_ID>"
```

Comment:
```suggestion
TIPTAP_APP_ID="<TIPTAP_APP_ID>"
```

Reply:
I played around with this a bit and realized that the reason it wasn't working for us when we were pairing is that a value that **is not** prefixed with `NEXT_PUBLIC_` is not available to the browser at all and is only available for server rendered components. In our case we need it in a component rendered on the client. I _think_ this is ok, at least at the moment. Looking at the devtools we can see the App ID as part of the URL our tiptap library is using for the WS messaging.  So it's exposed via the internals of Tiptap itself.

Reply:
Thanks for looking into this Caleb. If it's exposed anyway by normal TipTap operation, then I agree that it's ok to be a NEXT_PUBLIC_* variable.